### ISSUE TITLE
[cm11] [Smultron] Music playback pauses, can't be easily resumed, via Bluetooth

### ISSUE THREAD

**Issue by Pazu** [LegacyXperia/local_manifests#1039]
Running cm-11-20151117 on my ST15i with a LG HBS700 Bluetooth headset
with 2.1+EDR (Headset / Hands-Free / A2DP Profiles) connected, the
Apollo music player pauses playing any track after some time ranging from 20 seconds to a few minutes.

Music can be played indefinitely using regular wired headphones.

Logs generated immediately after playback failure are attached.

[bugreport.zip](https://github.com/LegacyXperia/local_manifests/files/164915/bugreport.zip)

**Comment by Pazu** [comment]
(This is playing M4A AAC files, by the way, not MP3, in case that's relevant.)

**Comment by Pazu** [comment]
Also reproduced with another Bluetooth device connected, this time a car audio system.

**Comment by realcatgirly** [comment]
try to Change build.prop line "lpa.decode=true" to "lpa.decode=false". worked for me on cm12.1(thanks to @MardonHH )

**Comment by Pazu** [comment]
@TomTomcrafter001 that, indeed, appears to have solved the issue. I guess this is because this chipset doesn't support (or well support) the DSP tunneling offered by KitKat's Low Power Audio (LPA). Is that accurate? If so, perhaps this should be the default setting for lpa.decode for these msm7x30 devices.

**Comment by realcatgirly** [comment]
@Pazu thanks, i didnt know that lpa means low power Audio. 

Issue solved, so Close it. Maybe a commit to set lpa.decode in buildprop to false for everyone.

**Comment by mikeNG** [comment]
lpa.decode fixes mp3 stuttering in normal playback, there is no easy way to fix both so for now it will stay as-is

**Comment by Pazu** [comment]
I guess since playback works fine in normal playback with lpa.decode=true, then my description of the issue can't be accurate. It's quite difficult to find any description of exactly what the lpa.decode setting controls.

**Comment by mikeNG** [comment]
lpa.decode enables LPA, which in turn fixes mp3 stuttering, but is buggy. There's no easy solution really for our ancient devices

**Comment by Pazu** [comment]
@mikeNG, thanks for the clarification. Would it be helpful for me to enable LPA verbose logging? (I noticed there is such a thing, in your change number 103550.) Can that be done in build.prop?

Also, I noticed some other typical build.prop settings relating to LPA (e.g., tunnel.decode=true and lpa.use-stagefright=true) weren't set. Is there a reason for that? I tried setting lpa.decode back to true and including those other two, but it didn't mitigate the Bluetooth playback issue.

Finally, is this the same basic issue as #881 ?
